mappers/refund.go

```diff
@@ -0,0 +1,35 @@
+package mappers
```

Comment:
Should this file be a transformer rather than a mapper? Just to keep consistent with the existing patterns.

Also add a test for this

Reply:
Please see answer to Peadar's query. Tests added.